common/src/writer.rs

```diff
@@ -62,7 +62,7 @@ impl<W: TerminatingWrite> TerminatingWrite for CountingWriter<W> {
 pub struct AntiCallToken(());
 
 /// Trait used to indicate when no more write need to be done on a writer
-pub trait TerminatingWrite: Write + Send {
+pub trait TerminatingWrite: Write + Send + Sync {
```

Comment:
Is this a requirement of `SimpleIndexWriter`?

Reply:
Yes it is required to be usable in Quickwit.